extensions/components/com_redcore/admin/views/translation/tmpl/modal-edit_fields.php

```diff
@@ -0,0 +1,192 @@
+<?php
```

Comment:
Right, that'd make sense. 
Since they'd both use the same layout, would it be fine to have them both look the same? Right now I'm using a horizontal design for the new feature (as it was suggested a long time ago), while I haven't touched the admin one at all.